### DISCUSSION TITLE
Source code for the "customized" example on the home page.

### DISCUSSION THREAD

**Discussion by Fawwaz-2009** [nick-keller/react-datasheet-grid#321] [Q&A]
I'm trying to find the source for the example below, I'm specially curious about the multi line text input. I tried using text area but the results weren't' great as the textarea input doesn't expand with input. I wonder if anyone has link the source code or an example with multiline text input example.

![image](https://github.com/nick-keller/react-datasheet-grid/assets/32019791/cc78be74-ff5c-48fd-94f7-0e6909e8f80e)

**Comment by nick-keller** [comment]
The source code is not available anymore. But feel free to add it to the doc if you re-create it!
